### ISSUE TITLE
Is that legal to use test set as validation set in voc0712.py?

### ISSUE THREAD

**Issue by truemanluo** [open-mmlab/mmdetection#6941]
I notice that the test set of voc2007 is utilized to validate during training process, is that legal to do this?
Thanks.
https://github.com/open-mmlab/mmdetection/blob/6c1347d7c0fa220a7be99cb19d1a9e8b6cbf7544/configs/_base_/datasets/voc0712.py#L47
